src/app/drive/components/ShareInviteDialog/ShareInviteDialog.tsx

```diff
@@ -70,158 +70,144 @@ const ShareInviteDialog = (props: ShareInviteDialog): JSX.Element => {
-                  >
-                    {({ selected }) => (
-                      <>
-                        <span>{translate('modals.shareModal.invite.editor')}</span>
-                        {selected ? <Check size={20} /> : null}
-                      </>
-                    )}
-                  </Listbox.Option>
-                  <Listbox.Option
-                    key="viewer"
-                    value="viewer"
-                    className="flex h-9 w-full cursor-pointer items-center justify-between space-x-3 rounded-lg py-2 px-3 text-base font-medium hover:bg-gray-5"
-                  >
-                    {({ selected }) => (
-                      <>
-                        <span>{translate('modals.shareModal.invite.viewer')}</span>
-                        {selected ? <Check size={20} /> : null}
-                      </>
-                    )}
-                  </Listbox.Option>
-                </Listbox.Options>
-              </div>
-            )}
-          </Listbox>
-        </form>
-        <div className="font-regular mt-1.5 text-xs text-gray-100">
-          {translate('modals.shareModal.invite.instructions')}
+      {usersToInvite.length != 0 && (
+        <div className="mt-4">
+          <h5 className="mb-2.5 text-lg font-medium">{translate('modals.shareModal.invite.listUsers')}</h5>
+          <ul>
+            {usersToInvite.map((user) => (
+              <li
+                key={user.email}
+                className="share-invite-user flex items-center justify-between border-b border-gray-5 py-2"
+              >
+                <div className="flex items-center">
+                  <Avatar src="" fullName={`${user.email}`} diameter={40} />
+                  <p className="ml-2.5">{user.email}</p>
+                </div>
+                <Listbox value={user.userRole} onChange={onEditRole}>
+                  {({ open }) => (
+                    <div className="relative">
+                      <Listbox.Button value={user.userRole} name={user.email}>
+                        <Button variant="secondary">
+                          <span className="capitalize">{user.userRole}</span>
+                          <CaretDown
+                            size={24}
+                            className={`${open ? 'rotate-180 transform transition' : 'rotate-0 transform transition'}`}
+                          />
+                        </Button>
+                      </Listbox.Button>
+                      <Listbox.Options className="absolute right-0 z-10 mt-1 w-40 transform whitespace-nowrap rounded-lg border border-gray-10 bg-white p-1 shadow-subtle transition-all duration-50 ease-out">
+                        <Listbox.Option
+                          key="editor"
+                          value="editor"
+                          className="flex h-9 w-full cursor-pointer items-center justify-start justify-between space-x-3 rounded-lg py-2 px-3 px-3 text-base font-medium hover:bg-gray-5"
```

Comment:
`justify-start justify-between` are contradicting each other, only use one.
`px-3` is repeated 2 times.